addons/mail/static/src/models/popover_view/popover_view.js

```diff
@@ -70,12 +144,55 @@ registerModel({
         channelInvitationForm: one2one('ChannelInvitationForm', {
             isCausal: true,
             readonly: true,
-            required: true,
         }),
         /**
          * States the OWL component of this popover view.
          */
         component: attr(),
+        /**
+         * If set, this popover view is owned by a composer view.
+         */
+        composerViewOwnerAsEmoji: one2one('ComposerView', {
+            inverse: 'emojisPopoverView',
+            readonly: true,
+        }),
+        /**
+         * Determines the record that is content of this popover view.
+         */
+        content: one2one('Model', {
+            compute: '_computeContent',
+        }),
+        /**
+         * Determines the class name for the component
+         * that is content of this popover view.
+         */
+        contentClassName: attr({
+            compute: '_computeContentClassName',
+            default: '',
+        }),
```

Comment:
maybe also required (at least with current code it looks always set), though for this one it would work and be acceptable to not be if we don't always want an extra class